src/main/target/common_post.h

```diff
@@ -91,6 +91,51 @@
 // normalize serial ports definitions
 #include "serial_post.h"
 
+#if !defined(USE_ACC_ADXL345) \
+    && !defined(USE_ACC_BMA280) \
+    && !defined(USE_ACC_LSM303DLHC) \
+    && !defined(USE_ACC_MMA8452) \
+    && !defined(USE_ACC_LSM303DLHC) \
+    && !defined(USE_ACC_MPU6000) \
+    && !defined(USE_ACC_MPU6050) \
+    && !defined(USE_ACC_MPU6500) \
+    && !defined(USE_ACC_SPI_MPU6000) \
+    && !defined(USE_ACC_SPI_MPU6500) \
+    && !defined(USE_ACC_SPI_MPU9250) \
+    && !defined(USE_ACC_SPI_ICM20602) \
+    && !defined(USE_ACC_SPI_ICM20649) \
+    && !defined(USE_ACC_SPI_ICM20689) \
+    && !defined(USE_ACCGYRO_BMI160) \
+    && !defined(USE_ACCGYRO_BMI270) \
+    && !defined(USE_ACC_SPI_ICM42605) \
+    && !defined(USE_ACC_SPI_ICM42688P) \
+    && !defined(USE_ACCGYRO_LSM6DSO) \
+    && !defined(USE_ACCGYRO_LSM6DSV16X) \
+    && !defined(USE_VIRTUAL_ACC)
+#error At least one USE_ACC device definition required
+#endif
```

Comment:
You should test for USE_ACC (it is  line 28 of src/main/sensors/acceleration_init.c).

```suggestion
#if defined(USE_ACC) \
    && !defined(USE_ACC_ADXL345) \
    && !defined(USE_ACC_BMA280) \
    && !defined(USE_ACC_LSM303DLHC) \
    && !defined(USE_ACC_MMA8452) \
    && !defined(USE_ACC_LSM303DLHC) \
    && !defined(USE_ACC_MPU6000) \
    && !defined(USE_ACC_MPU6050) \
    && !defined(USE_ACC_MPU6500) \
    && !defined(USE_ACC_SPI_MPU6000) \
    && !defined(USE_ACC_SPI_MPU6500) \
    && !defined(USE_ACC_SPI_MPU9250) \
    && !defined(USE_ACC_SPI_ICM20602) \
    && !defined(USE_ACC_SPI_ICM20649) \
    && !defined(USE_ACC_SPI_ICM20689) \
    && !defined(USE_ACCGYRO_BMI160) \
    && !defined(USE_ACCGYRO_BMI270) \
    && !defined(USE_ACC_SPI_ICM42605) \
    && !defined(USE_ACC_SPI_ICM42688P) \
    && !defined(USE_ACCGYRO_LSM6DSO) \
    && !defined(USE_ACCGYRO_LSM6DSV16X) \
    && !defined(USE_VIRTUAL_ACC)
#error At least one USE_ACC device definition required
#endif
```